lib/model/folderstate.go

```diff
@@ -38,7 +38,7 @@ func (s folderState) String() string {
 }
 
 type stateTracker struct {
-	folder string
+	folderId string
```

Comment:
This may be a valid refactor when there is ambiguity (often there is not, as a folder is represented by it's ID). Regardless, "ID" in this code base is spelled with capitals, so it'll be `folderID`.